vividus-to-xray-exporter/src/main/java/org/vividus/xray/exporter/XrayExporter.java

```diff
@@ -140,8 +140,10 @@ private void publishErrors()
                                 .append("Error: ").append(errorEntry.getError()).append(lineSeparator());
                 });
                 return errorBuilder.toString();
-            }).log("Errors:{}{}");
+            }).log("EXPORT FAILED:{}{}");
+            return;
         }
+        LOGGER.atInfo().log("EXPORT SUCCESSFUL");
```

Comment:
why do you need uppercase?

Reply:
I leverage upper case stuff from Gradle, imho looks goos :)